[STRATOS-1166] - Python Cartridge Agent doesn't detect member initialized status

chamilad opened the pull request:
The Member class is updated with to reflect the changes on the messaging model.

imesh commented:
@chamilad Would you mind explaing the changes you have done in this pull request in few sentences? I see considerable amount of changes.
Thanks

chamilad commented:
@imesh I've fixed the error where the status of the Member object in the topology is not being properly serialized. member.status is now set to the latest of the status stack from MemberStateManager. The other changes to the Member class in the messaging component was also reflected in the Member class in the Python code. Additionally I've added a unit test to verify serialization of the CompleteTopologyEvent and refactored old unit tests to re enable unit testing for Python agent.

imesh commented:
Great! Thanks for the explanation! I have now merged this pull request to the master branch.
Thanks
